certbot/compat.py

```diff
@@ -148,3 +148,29 @@ def compare_file_modes(mode1, mode2):
     # Windows specific: most of mode bits are ignored on Windows. Only check user R/W rights.
     return (stat.S_IMODE(mode1) & stat.S_IREAD == stat.S_IMODE(mode2) & stat.S_IREAD
             and stat.S_IMODE(mode1) & stat.S_IWRITE == stat.S_IMODE(mode2) & stat.S_IWRITE)
+
+WINDOWS_DEFAULT_FOLDERS = {
+    'config': 'C:\\letsencrypt',
+    'workdir': 'C:\\letsencrypt\\lib',
```

Comment:
I did that.

Reply:
Again, I should have been more clear in my comment here.

I wasn't referring the code structure. In fact, I prefer the fields being global constants that can be easily seen in the codebase.

All I meant is that what's stored in `--work-dir` on disk are files that the user should never touch. It's used for a few things, but primarily for configuration backups written in a custom format that's pretty fragile to user modifications.

With this information, if you can think of a better or more conventional place on the Windows filesystem for a program to store data like this, I'd be interested in hearing about it and maybe moving `--work-dir` there to prevent problems, but if you're not `Certbot\lib` is fine with me.

Reply:
All right I got it! Yes, I will rethink about that then. Certainly ProgramData is a good place, as it is hidden and made for application cache. Could be the solution if is is available for all Windows versions that are targeted.

Reply:
Yeah let me know what you think! I'm not super concerned about putting the files in the `Certbot` folder, it was just something I thought we might want to consider.

@joohoi, if you or your friends are interested and have thoughts on this issue as well, I'd love to hear them.

Reply:
After discussions we had on IRC, work dir will stay in `C:\Certbot`, because its content is critical for some Certbot functionalities. Any other place (like `AppData` or `ProgramData` will make theses files more likely to be deleted accidentally).